src/types.ts

```diff
@@ -57,15 +58,16 @@ class DownloadRelease {
   }
 
   static fromTag(id: number, tag: String): DownloadRelease | undefined {
-    const pattern = /download-latest-v(\d+)/
+    const pattern = /^download-latest(?:-v(\d+))?$/
```

Comment:
Is the `(?:...` really necessary since you have a `?` after the outer group?

Reply:
The `?` of `(?:` is not a quantifier but an option to turn the `(` as a non capturing group.
As I don't care about capturing `-vX` but just only `X`, I disabled it (by I had to have some parenthesis to define the scope of the second `?`).
Am I clear? (I does not sound while I am typing it 😓 )

Reply:
Ah of course. That makes sense. Thanks for the explanation.